### PR TITLE
Issue/42 export csv fix

### PR DESCRIPTION
**Pull request by w-strzebonski**

### CODE REVIEW

**Comment by tomaszmadeyski** [comment]
See my [comment in the issue](https://github.com/fortedigital/EpiserverRedirects/issues/42#issuecomment-818620951)

**Comment by w-strzebonski** [comment]
See my [comment in the issue](https://github.com/fortedigital/EpiserverRedirects/issues/42#issuecomment-819384458)

File: AlloyDemoKit/modules/_protected/Forte.EpiserverRedirects/ClientResources/RedirectsMenu/RedirectsMenu.html [hunk 1]
```diff
@@ -12,8 +12,9 @@ <h1 class="EP-prefix">
 
             <div class="redirect-importer">
                 <h2 class="redirect-importer-header">Import redirects from CSV file</h2>
-                <p class="redirect-importer-description">Select a file, and click "upload". File shoud be in CSV format 
-                    and consist of two columns: old URL and new URL.
+                <p class="redirect-importer-description">Select a file, and click "upload". File shoud be in CSV format
```
Comment (tomaszmadeyski) [inline AlloyDemoKit/modules/_protected/Forte.EpiserverRedirects/ClientResources/RedirectsMenu/RedirectsMenu.html:15]: it says `click upload` but the button label is actually `submit`, not `upload`

File: EpiserverRedirects/ClientResources/RedirectsMenu/RedirectsMenu.html [hunk 2]
```diff
@@ -12,8 +12,9 @@ <h1 class="EP-prefix">
 
             <div class="redirect-importer">
                 <h2 class="redirect-importer-header">Import redirects from CSV file</h2>
-                <p class="redirect-importer-description">Select a file, and click "upload". File shoud be in CSV format 
-                    and consist of two columns: old URL and new URL.
+                <p class="redirect-importer-description">Select a file, and click "upload". File shoud be in CSV format
```
Comment (tomaszmadeyski) [inline EpiserverRedirects/ClientResources/RedirectsMenu/RedirectsMenu.html:15]: same applies here (upload vs submit)

**Review by tomaszmadeyski — APPROVED** [review]

**Pull request merged**